packages/core/admin/admin/src/core/apis/tests/content-manager.test.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable check-file/filename-naming-convention */
+import { ContentManagerPlugin, PanelComponent } from '../content-manager';
+
+describe('content-manager', () => {
+  describe('config', () => {
+    it("should export the a config shape to pretend it's a plugin", () => {
+      const plugin = new ContentManagerPlugin();
+
+      expect(plugin.config).toMatchInlineSnapshot(`
+        {
+          "apis": {
+            "addEditViewSidePanel": [Function],
+            "getEditViewSidePanels": [Function],
+          },
+          "id": "content-manager",
+          "name": "Content Manager",
+        }
+      `);
+    });
+
+    /**
+     * This test will help you ensure you don't remove any API
+     * because they're public facing. You can of course add new ones.
+     */
+    it('should export only these APIs', () => {
+      const plugin = new ContentManagerPlugin();
+
+      expect(Object.keys(plugin.config.apis ?? {})).toMatchInlineSnapshot(`
+        [
+          "addEditViewSidePanel",
+          "getEditViewSidePanels",
+        ]
+      `);
+    });
+  });
+
+  describe('addEditViewSidePanel', () => {
+    it('should let users add a panel description as an array', () => {
+      const plugin = new ContentManagerPlugin();
+
+      expect(plugin.editViewSidePanels).toMatchInlineSnapshot(`
+        [
+          [Function],
+          [Function],
+        ]
+      `);
+
```

Comment:
is a snapshot useful here? I feel like you could just measure the array length

Reply:
Yeah these tests could probably be better, we probably need to assert that certain panels have left too, i'll update it.